### DISCUSSION TITLE
MU format

### DISCUSSION THREAD

**Discussion by mamonulabs** [ohmtech-rdi/eurorack-blocks#405] [Ideas]
Hi . Really interesting project!

For the sake of discussion...while this project is called eurorack-blocks I was wondering how difficult it would be to have an option for creating modules for large-format 5U/MU modular systems ?

Having a quick look the dimensions of patch.init for example would fit a 1MU width panel... 
so sizewise ..at least width wise it fits. Only size and location of the jacks (usually Switchcraft) needs to change

There is also the issue of +-15V instead of 12V but this can be solved quite easily with the inclusion of a add-on like
https://oshpark.com/shared_projects/IMyjiSay 

Sorry if this is extremely out of scope... just wondering if this could be possible thats all :)
I could provide some more details regarding the 5U/MU format if that would be of help...

**Comment by ohmtech-rdi** [comment]
Hi! Thanks for the nice words!

The project is called Eurorack-blocks, but we want to change it to ERB (like the logo), exactly for this reason. The electrical standard goes past Eurorack, in particular for sound modules, which is much more attractive to a portion of potential users.

From a mechanical point of view, there are only a few points where 3U, 128.5mm is assumed. From a Erbui language perspective we could introduce a `height` keyword for the MU format.

From an electrical point of view, we introduced [inline custom boards](https://github.com/ohmtech-rdi/eurorack-blocks/pull/385) lately, so theoretically, you can do whatever you want at the board level. I don't know anything about the MU format, but what are gate, CV and audio levels look like?

Finally, we will introduce soon the concept of [semantic styles](https://github.com/ohmtech-rdi/eurorack-blocks/discussions/398), which would allow to adapt (to a certain extent) to specific parts for the MU format.

Maybe we should have a call on Slack and talk about it. What do you think?

**Reply by ohmtech-rdi** [reply]
Actually the [Daisy Patch Submodule (on which Eurorack-blocks relies) supports ±17V power input directly](https://static1.squarespace.com/static/58d03fdc1b10e3bf442567b8/t/621eaf2cdbf1c54baf279bdd/1646178095682/ES_Patch_SM_datasheet_v1.0.2.pdf) so you are in luck!

I've looked a bit at the format and it seems it's more or less the same as Eurorack. I couldn't find the official electrical specifications for 5U/MU but they seem to be the same as MOTM? Also I've corrected a mistake on the MOTM wikipedia page (they were saying the audio signal is ±10V, but that's 10V peak-to-peak right?
